### ISSUE TITLE
bytestring is used when django User created

### ISSUE THREAD

**Issue by romchi** [robgolding/django-radius#18]
Hi,

When I`m get authorization, my username looks b'UserName'

https://github.com/robgolding/django-radius/blob/5b90f4ae4cbd680988197386d882491317e6b20c/radiusauth/backends/radius.py#L149

Fix look:
```
    def get_django_user(self, username, password=None):
        """
        Get the Django user with the given username, or create one if it
        doesn't already exist. If `password` is given, then set the user's
        password to that (regardless of whether the user was created or not).
        """
        try:
            if isinstance(username, basestring):
                user = User.objects.get(username=username.decode("utf-8"))
            else:
                user = User.objects.get(username=username)
        except User.DoesNotExist:
            if isinstance(username, basestring):
                user = User(username=username.decode("utf-8"))
            else:
                user = User(username=username)

        if password is not None:
            if isinstance(username, basestring):
                user.set_password(password.decode("utf-8"))
            else:
                user.set_password(password)
            user.save()

        return user
```

**Comment by Azendale** [comment]
@romchi I think I might have fixed this in my branch.

**Comment by snatch2013** [comment]
Yes, the same problem.

**Comment by robgolding** [comment]
Fixed in #19. Released v1.4.0 with that PR incorporated.
